docs/program/apis/data-client.md

```diff
@@ -703,3 +703,97 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 {{% /tab %}}
 {{< /tabs >}}
+
```

Comment:
Don't forget to also update these in `/static/include/services/apis/data-client.md`

Reply:
Thank you!